.env.development

```diff
@@ -0,0 +1,24 @@
+VITE_UPLOADER=tus
+
+#### To customize this, create a .env.dev.local file that overrides those values
```

Comment:
nit: it's `.env.development` but `.env.dev.local` (instead of both `dev` or both `development`)